servers/ups-wildfly/src/main/webapp/config/admin-ui-keycloak.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
It's replace by a call to `/rest/config/KeycloakConfigurationEndpoint.java` that returns the json config